runbooks/workflow/upgrade-metabase.md

```diff
@@ -0,0 +1,48 @@
+# Upgrading Metabase
+
+Metabase upgrades happen in two phases: validation and release.
+
+## Validation
+
+### Prerequisites
+
+- Lens (https://k8slens.dev)
+- Restic (https://restic.net)
+- A copy of the secrets for the GCloud service account
+
+### Service Credentials
+
+1. Log into GCloud: `gcloud auth login`
+2. List the clusters you can access: `gcloud container clusters list`
+3. Pull Kubernetes credentials for the `data-infra-apps` cluster to your local machine: `gcloud container clusters get-credentials data-infra-apps --location us-west1`
+
+### Replicating the database
+
+1. Retrieve environment variables for one of the `metabase-test` `postgresql-backup` pods
+2. Set the `GOOGLE_APPLICATION_CREDENTIALS`, `GOOGLE_PROJECT_ID`, `RESTIC_REPOSITORY` and `RESTIC_PASSWORD` environment variables in a shell
+3. List available Restic snapshots using `restic snapshots` and note the latest SHA and filename
+4. Download the latest snapshot `restic dump -t <OUTPUT> <SHA> <FILENAME>`
+5. Uncompress the dump and make sure it is a complete dump including both schema and data
+
+### Restoring the database
+
+1. Visit the test instance at https://metabase-test.k8s.calitp.jarv.us/ to see that login and reporting work as expected
+2. Connect to the database instance `kubectl -n metabase-test exec -it database-0 -- bash`
+3. Ensure that `psql` can connect: `psql -U admin postgres`
+4. Scale the `metabase` ReplicaSet to zero
```

Comment:
We need to be explicit about doing this inside the `metabase-test` namespace. We can just include a pastable kubectl command for doing it that has the namespace baked in

Reply:
@themightychris Please add the command here

Reply:
```suggestion
4. Scale the Metabase deployment down to 0 replicas to disconnect the database from use:

    ```bash
    kubectl -n metabase-test scale deployment metabase --replicas=0 
    ```
```